pkgs/development/python-modules/verilogae/default.nix

```diff
@@ -0,0 +1,100 @@
+{
+  buildPythonPackage,
+  fetchFromSourcehut,
+  stdenv,
+  llvmPackages,
+  rustc,
+  patchelf,
+  python,
+  setuptools-rust,
+  cargo,
+  clang,
+  rustPlatform,
+  lib,
+  zlib,
+  ncurses,
+  libxml2,
+  pkg-config,
+  pkgs,
+}:
+
+buildPythonPackage rec {
+  pname = "verilogae";
+  version = "0.9-beta-8";
+
+  src = fetchFromSourcehut {
+    owner = "~dspom";
+    repo = "OpenVAF";
+    rev = "b800153e84a8a640b07048d580fabafc25bbc841";
+    hash = "sha256-NKiJrXnxwghNYtKL4s3YjvQd/r3QWe5saCYp4N4aQ8w=";
+  };
+
+  format = "other";
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
This is Import From Derivation and won't evaluate on hydra. Try to upload Cargo.lock instead.